plugins/test/chromatic_correction_test.py

```diff
@@ -0,0 +1,87 @@
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+import unittest
+
+import numpy
+from odemis import model
+
+from odemis.dataio.tiff import read_data
+from odemis.util.spot import MaximaFind
+from odemis.util.transform import AffineTransform, alt_transformation_matrix_to_implicit
+from plugins.chromatic_correction import (ACTUAL_BEAD_DIAMETER, SCALE,
+                                          BEAD_QUANTITY, get_chromatic_correction_dict)
+
+
+class TestChromaticAberrationCorrection(unittest.TestCase):
+    """
+    Test the Chromatic Aberration Correction functions used in the plugin
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        cls.das = []
+        for input_file_path in ['FOV1_green.tif', 'FOV1_blue.tif']:
+            cls.das.append(read_data(input_file_path))
+
+    def test_get_chromatic_correction_dict(self):
+        max_dist = ACTUAL_BEAD_DIAMETER * SCALE  # size of bead in the image
+        ref = self.das[0][0]
+        im = self.das[1][0]
+        im.metadata.update({model.MD_PIXEL_SIZE: (0.07738095984842844 * 1e-6, 0.07738095984842844 * 1e-6)})
+
+        # convert m to pixel units
+        max_dist_pxl = int(max_dist / im.metadata[model.MD_PIXEL_SIZE][0])
+
+        # Find transformation values per channel
+        points_ref = MaximaFind(ref, BEAD_QUANTITY, max_dist_pxl)
+        points_im = MaximaFind(im, BEAD_QUANTITY, max_dist_pxl)
+        _ = get_chromatic_correction_dict(im, points_ref, points_im)
+
+        ## Validation
+        affine = AffineTransform.from_pointset(points_ref, points_im)
+        mat = numpy.eye(3)
+        mat[:2, :2] = affine.matrix
+        mat[:2, 2] = affine.translation
+        ref_mat = numpy.eye(3)
+        mat = ref_mat @ mat
+
+        # Translation is the last column
+        tx = mat[0, 2]
+        ty = mat[1, 2]
+        pixel_size = im.metadata[model.MD_PIXEL_SIZE]
+        # Projection image y axis is upside down, add negative sign
+        translation_cor = [tx * pixel_size[0], -ty * pixel_size[1]]
+
+        # Scale is the norm of the first two columns
+        sx = numpy.linalg.norm(mat[:2, 0])
+        sy = numpy.linalg.norm(mat[:2, 1])
+        scale_cor = [1 / sx, 1 / sy]
+
+        # Compute rotation
+        rotation_cor = numpy.arctan2(mat[1, 0] / sx, mat[0, 0] / sx)
+
+        # Shear correction
+        a, b = mat[0, 0], mat[0, 1]
+        c, d = mat[1, 0], mat[1, 1]
+        shear_cor = numpy.arctan2(a * c + b * d, scale_cor[0] * scale_cor[1])
```

Comment:
This whole block is the code from `extract_transformation_parameters()`. Of course, the result will be equal. 
I think that instead of check the exact values, it'd make more sense to just check that the metadata looks correct (ie, it's present, and it's a float or a tuple of 2 floats).

As you have a specific dataset (FOV1 green & blue), you can also compare the values to a set of hardcoded values.